### ISSUE TITLE
[UX] Rich-text Image/text flow.

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#987]
It would be really great UX to make it so that with _each_ image upload, there is a set of options in the form of buttons/icons to set the flow of the text left/right/above/below/around the image. Here's how [Craft CMS](https://buildwithcraft.com) does it:

![craft_cms-wysiwyg_editor-image_text_flow](https://cloud.githubusercontent.com/assets/2423362/8096170/6bd565be-1017-11e5-89cb-2d8e32808088.png)

**Comment by quicksketch** [comment]
This is interesting because it sort of relates to the work @BWPanda did. We added formatter settings to make it so that images could be aligned left/right, but it's on the entire content type (see https://github.com/backdrop/backdrop-issues/issues/379).

I'm not sure how we would implement this in our current paradigm. Formatters and data are generally entirely separate. It's still an interesting idea, but I have no idea how to go about it.

**Comment by klonos** [comment]
Yeah, I just saw it and thought it was cool. Can't help with figuring out a possible implementation though. Sorry.
